contracts/TokenCrowdsale.sol

```diff
@@ -54,16 +54,15 @@ contract TokenCrowdsale is MintedCrowdsale, FinalizableCrowdsale {
      require(_uncappedOpeningTime <= _closingTime);
      require(_uncappedOpeningTime >= _openingTime);
 
-     require(_icoTotalCap > 0);
-
```

Comment:
it would be good to add check that icopercentage is equal to ico-cap + private-cap percentage of total